packages/@ember/-internals/metal/lib/properties.ts

```diff
@@ -4,13 +4,11 @@
 
 import { Meta, meta as metaFor, peekMeta, UNDEFINED } from '@ember/-internals/meta';
 import { setWithMandatorySetter } from '@ember/-internals/utils';
-import { EMBER_METAL_TRACKED_PROPERTIES } from '@ember/canary-features';
 import { assert } from '@ember/debug';
 import { DEBUG } from '@glimmer/env';
 import { Decorator } from './decorator';
 import { descriptorForProperty, isClassicDecorator } from './descriptor_map';
 import { revalidateObservers } from './observer';
-import { overrideChains } from './property_events';
 
 export type MandatorySetterFunction = ((this: object, value: any) => void) & {
```

Comment:
I believe this can be removed too, since the mandatory setter stuff has all been pulled into `utils`